qml/pages/InitializationPage.qml

```diff
@@ -147,7 +147,7 @@ Page {
 
                 TextField {
                     id: phoneNumberTextField
-                    placeholderText: "Use the international format, e.g. +4912342424242"
+                    placeholderText: qsTr("Use the international format, e.g.") + "+4912342424242"
```

Comment:
If you want to split the string, maybe ".arg()" is the better way?
```suggestion
                    placeholderText: qsTr("Use the international format, e.g. %1").arg("+4912342424242")
```

Reply:
ok